templates/base.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="{{ url_for('static', filename='../MUNI/css/style-fi.css') }}">
```

Comment:
`css/MUNI/css/style-fi.css` should be right way because by 'static' you are defining folder and then path in it. After structure change it should be `MUNI/css/style-fi.css`
```suggestion
    <link rel="stylesheet" href="{{ url_for('static', filename='css/MUNI/css/style-fi.css') }}">
```